controllers/statusreport/statusreport_adapter.go

```diff
@@ -72,7 +72,10 @@ func NewAdapter(context context.Context, snapshot *applicationapiv1alpha1.Snapsh
 // which (indirectly) triggered its execution.
 func (a *Adapter) EnsureSnapshotTestStatusReportedToGitProvider() (controller.OperationResult, error) {
 	if gitops.IsSnapshotCreatedByPACPushEvent(a.snapshot) {
+		a.logger.Info("Snapshot is created by PAC push event, skipping 'EnsureSnapshotTestStatusReportedToGitProvider'")
 		return controller.ContinueProcessing()
+	} else {
+		a.logger.Info("Snapshot is not created by PAC push event, running 'EnsureSnapshotTestStatusReportedToGitProvider'")
```

Comment:
There is log.info later showing which reporter will be used, do we really need this log message?

Reply:
This was a debugging statement I forgot to remove.  Fixed.